L/Libbpf/bundled/patches/musl-add-stddef.patch

```diff
@@ -0,0 +1,22 @@
+From bf61d9cb6472364d96dbb5cf5abe5715d9a4d7ff Mon Sep 17 00:00:00 2001
```

Comment:
Try to upstream the patch? :slightly_smiling_face:

Reply:
Yeah, this would be a good first Linux contribution :slightly_smiling_face: